app/lib/services/autocomplete_itt_providers.rb

```diff
@@ -5,7 +5,18 @@ def self.as_open_structs
     end
   end
 
+  def self.as_autocomplete_options
+    all_names.map do |legal, operating|
```

Comment:
The `all_names` method name is little misleading, I would simply use query here:
```
IttProvider.pluck(:legal_name, :operating_name).map do ...
```